webapp/design_assembly.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import os
+import hashlib
+
+from datetime import datetime, timedelta
+from dateutil import parser
+from flask import Blueprint, render_template, request
+
+# from webapp.sso import login_required
+
+DEPLOYMENT_ID = os.getenv(
```

Comment:
Do we want to control this through environment variables or through code? I'm not sure it should be both, I think that makes it confusing to work out the state of any specific environment. Personally I think the team has much more visibility over code than we do over the Kubernetes secrets, so I'd probably suggest we just hard-code this and if we need to change it we do so with a pull request. Thoughts?

Reply:
This is down to my not being too comfortable with Python, we based a lot of this off the way /canonicool is [set up](https://github.com/canonical-web-and-design/webteam.canonical.com/blob/main/webapp/canonicool.py), it helped us get to the meat of things faster during the hack day - but I see what you mean, so I've updated this to be hardcoded.